### ISSUE TITLE
aws_elasticsearch_domain shows no price for ebs storage

### ISSUE THREAD

**Issue by pkudrynskyi** [infracost/infracost#2623]
aws_elasticsearch_domain resource shows no price changes if we modify ebs storage. But according to AWS documentation there're some charges for provisioneing ebs gp3 storage. From the output I just see that **1 x aws_elasticsearch_domain_saml_options not supported yet** but that's not a case. Infracost version is v0.10.28. is it a bug?

``` aws_elasticsearch_domain.es_prod |
Instance (on-demand, r6g.large.elasticsearch)          2,190  hours           $442.38 
Storage (gp3)                                          700  GB                $0.00 
Dedicated master (on-demand, m6g.large.elasticsearch)  2,190  hours      $335.07 
UltraWarm instance (on-demand, ultrawarm1.medium.elasticsearch)  1,460  hours      $414.64  


time="2023-08-16T16:24:01+03:00" level=warning file="github.com/infracost/infracost/internal/prices/prices.go:104" func=github.com/infracost/infracost/internal/prices.setCostComponentPrice msg="No products found for aws_elasticsearch_domain.es_prod Storage (gp3), using 0.00"
time="2023-08-16T16:24:01+03:00" level=warning file="github.com/infracost/infracost/internal/prices/prices.go:104" func=github.com/infracost/infracost/internal/prices.setCostComponentPrice msg="No products found for aws_elasticsearch_domain.es_prod Storage (gp3), using 0.00" ```

**Comment by aliscott** [comment]
Thanks for reporting @pkudrynskyi 🙏! The GP3 issue looks like a bug.

For `aws_elasticsearch_domain_saml_options` Infracost doesn't support that resource. We maintain a list of [supported resources here](https://www.infracost.io/docs/supported_resources/aws). If you'd like to request support for that resource please create a separate issue for it.

**Comment by hugorut** [comment]
@pkudrynskyi we've merged a fix for this into master. It will go out with the next release. I'll let you know here when that is available. Alternatively, if you use our GitHub or Gitlab app integration this functionality will already be available. Thanks.

**Comment by aliscott** [comment]
@pkudrynskyi, @erpel this is now released in [Infracost v0.10.29](https://github.com/infracost/infracost/releases/tag/v0.10.29).
